### ISSUE TITLE
Any chance of adding a context menu to edit bookmarks?

### ISSUE THREAD

**Issue by rafaelgssa** [honsiorovskyi/open-url-in-container#66]
It would be great if there was a context menu to edit bookmarks, so that we can add or change the container without having to manually edit the URL.

**Comment by honsiorovskyi** [comment]
Thank for your interest @rafaelgssa! Do you mean to add an item to the bookmark's context menu to be able to assign a specific container to it (e.g. to get bookmark URL and transform it to a "containerized" URL)?

**Comment by rafaelgssa** [comment]
Yes. It would preferably show a dropdown with all of the available containers, so that we can select which container to assign it to. Kind of how my extension [Container Bookmarks](https://addons.mozilla.org/en-US/firefox/addon/container-bookmarks) does it:

![](https://addons.cdn.mozilla.net/user-media/previews/full/223/223253.png?modified=1565029940)

**Comment by honsiorovskyi** [comment]
Thanks, I'll try to do something that would cover this use case :+1:

**Comment by Roy-Orbison** [comment]
A bookmarklet to tide us over:
```
javascript:(function(c,u)%7Bc&&(c=c.replace(/%5E%5Cs+%7C%5Cs+$/g,''))&&(u=prompt('URL?'))&&(u=u.replace(/%5E%5Cs+%7C%5Cs+$/g,''))&&prompt('Contained%20URL:','ext+container:name='+encodeURIComponent(c)+'&url='+encodeURIComponent(u))%7D)(prompt('Container%20name?','Personal'))
```
Replace the default container name at the end with whatever you want, but make sure it's a percent-encoded, valid JS string literal.

**Usage:** click bookmark, Optionally edit name, hit Enter, Ctrl + V to paste URL, hit Enter, Ctrl + C to copy contained URL, hit Esc/Enter.
